src/main/scala/no/ndla/learningpathapi/service/UpdateServiceComponent.scala

```diff
@@ -118,10 +118,9 @@ trait UpdateServiceComponent {
             existing.copy(
               status = status,
               lastUpdated = clock.now()))
-
           updatedLearningPath.isPublished match {
             case true => searchIndexService.indexLearningPath(updatedLearningPath)
-            case false => searchIndexService.deleteLearningPath(updatedLearningPath)
+            case false => if(existing.isPublished) searchIndexService.deleteLearningPath(updatedLearningPath)
```

Comment:
Her vil det jo komme inn en Scala match error ved (false, false). Er det ikke da bedre å feks ta i bruk en if else som for eksempel denne:
`          if (updatedLearningPath.isPublished) {
            searchIndexService.indexLearningPath(updatedLearningPath)
          } else if (existing.isPublished) {
            searchIndexService.deleteLearningPath(updatedLearningPath)
          }`